### PR TITLE
Port the gather-easyfix script to py3 and run black on it

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

File: gather_easyfix.py [hunk 1]
```diff
@@ -32,11 +32,11 @@
...
 import re
-import urllib2
-import xmlrpclib
+try:
+    from urllib2 import urlopen
+except:
+    from urllib.request import urlopen
 from bugzilla.rhbugzilla import RHBugzilla
-import fedora.client
-from kitchen.text.converters import to_bytes
```
Comment (hroncok) [inline gather_easyfix.py]: +points for removing kitchen

**Review by cverna — APPROVED** [review]
:+1:

**Comment by frenzymadness** [comment]
I can confirm that the tool works with Python 3.7 and the latest version of its dependencies.

Good job!

**Comment by pypingou** [comment]
Thanks :)

**Pull request merged**